orderbook/src/api.rs

```diff
@@ -74,6 +76,8 @@ pub fn handle_all_routes(
     routes_with_labels.recover(handle_rejection).with(cors)
 }
 
+pub type ApiReply = warp::reply::WithStatus<warp::reply::Json>;
```

Comment:
I think there are some modules that already return `WithSTatus` explicitely. Can we maybe change those to use this type alias:

https://github.com/gnosis/gp-v2-services/blob/02eeb54727eccbff521cc08321cb21189d360533/orderbook/src/api.rs#L129

Reply:
fixed